Master built broken after merge

**Problem description**
The master build broke after a merge of a PR that has been lingering around for some time (#2817).

The compilation error is the following:

```
controller:compileJava/home/travis/build/pravega/pravega/controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java:242: error: method authenticateExecuteAndProcessResults in class ControllerServiceImpl cannot be applied to given types;
        authenticateExecuteAndProcessResults(v -> checkAuthorization(request.getStreamInfo().getScope() + "/" +
        ^
  required: Supplier<String>,Function<String,CompletableFuture<T#1>>,StreamObserver<T#1>
  found: (v)->check[...]DATE),()->contro[...]ld()),StreamObserver<StreamCutValidityResponse>
  reason: cannot infer type-variable(s) T#1,U
    (argument mismatch; incompatible parameter types in lambda expression)
  where T#1,U,T#2 are type-variables:
    T#1 extends Object declared in method <T#1>authenticateExecuteAndProcessResults(Supplier<String>,Function<String,CompletableFuture<T#1>>,StreamObserver<T#1>)
    U extends Object declared in method <U>thenApply(Function<? super T#2,? extends U>)
    T#2 extends Object declared in class CompletableFuture
/home/travis/build/pravega/pravega/controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java:242: error: cannot find symbol
        authenticateExecuteAndProcessResults(v -> checkAuthorization(request.getStreamInfo().getScope() + "/" +
                                                  ^
  symbol:   method checkAuthorization(String,Permissions)
  location: class ControllerServiceImpl
2 errors
 FAILED
```

**Problem location**
Build, compilation.

**Suggestions for an improvement**
The problem has been caused by a breaking change that was merged between the verification of PR #2817 and it being merged. Fix the compilation error.

--- a/controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java
+++ b/controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java
@@ -239,9 +239,9 @@ public class ControllerServiceImpl extends ControllerServiceGrpc.ControllerServi
     public void isStreamCutValid(Controller.StreamCut request, StreamObserver<Controller.StreamCutValidityResponse> responseObserver) {
         log.info("isStreamCutValid called for stream {}/{} streamcut {}.", request.getStreamInfo().getScope(),
                 request.getStreamInfo().getStream(), request.getCutMap());
-        authenticateExecuteAndProcessResults(v -> checkAuthorization(request.getStreamInfo().getScope() + "/" +
+        authenticateExecuteAndProcessResults(() -> this.authHelper.checkAuthorizationAndCreateToken(request.getStreamInfo().getScope() + "/" +
                         request.getStreamInfo().getStream(), AuthHandler.Permissions.READ_UPDATE),
-                () -> controllerService.isStreamCutValid(request.getStreamInfo().getScope(),
+                delegationToken -> controllerService.isStreamCutValid(request.getStreamInfo().getScope(),
                         request.getStreamInfo().getStream(),
                         request.getCutMap())
                         .thenApply(bRes -> Controller.StreamCutValidityResponse.newBuilder().setResponse(bRes).build()),

